pandas/tests/groupby/aggregate/test_aggregate.py

```diff
@@ -866,6 +866,57 @@ def test_agg_namedtuple(self):
         expected = df.groupby("A").agg(b=("B", "sum"), c=("B", "count"))
         tm.assert_frame_equal(result, expected)
 
+    def n_between(self, ser, low, high, **kwargs):
+        return ser.between(low, high, **kwargs).sum()
+
+    def test_namedagg_args(self):
+        df = DataFrame({"A": [0, 0, 1, 1], "B": [-1, 0, 1, 2]})
```

Comment:
Can you add `# https://github.com/pandas-dev/pandas/issues/58283` as the first line in each of these tests.